src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -199,7 +199,9 @@ protected ExtensionsRunner(Extension extension) throws IOException {
             // initialize SDKClient action map
             initializeSdkClient();
             // store REST handlers in the registry
-            for (ExtensionRestHandler extensionRestHandler : ((ActionExtension) extension).getExtensionRestHandlers()) {
+            for (ExtensionRestHandler extensionRestHandler : ((ActionExtension) extension).getExtensionRestHandlers(
+                injector.getInstance(ExtensionsRunner.class)
```

Comment:
What benefit does this give us over simply passing `this` here, if we do keep this?